### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by malaykhakhar**

### CODE REVIEW

File: components/Cards/card.html [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        <div class="container-card">
+          <div class="card vertical-flex card-vertical text-card">
+            <div class="text-card-title">Title</div>
+            <div class="text-card-desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur, ipsam excepturi, fugit molestias est repellat rem exercitationem, ab perferendis tempore repellendus iusto totam qui optio illo rerum iste eveniet placeat!
+            </div>
+            <button class="text-card-button">Read More</button>
+          </div>
+        </div>
+
+        <iframe
```
Comment (AshutoshM29) [inline components/Cards/card.html:254]: It's my opinion, instead of adding the entire code snippet at once you can add below the respective component

**Review by AshutoshM29 — APPROVED** [review]
Rest looks good to merge!!

File: components/Cards/card.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.card-img {
+  flex-basis: 20rem;
+}
+
+.card-img img {
+  max-width: 100%;
+  max-height: 100%;
+  /* object-fit: contain; */
```
Comment (gauravsinhaweb) [inline components/Cards/card.css]: remove comments

**Review by gauravsinhaweb — APPROVED** [review]
good to merge after changes.

**Pull request merged**